packages/dd-trace/src/appsec/telemetry/verbosity.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const Verbosity = {
+  OFF: 0,
+  MANDATORY: 1,
+  INFORMATION: 2,
+  DEBUG: 3
+}
+
+const telemetryVerbosity = process.env.DD_IAST_TELEMETRY_VERBOSITY
```

Comment:
With this approach we are not allowing to configure telemetry verbosity via [TracerOptions](https://datadoghq.dev/dd-trace-js/interfaces/traceroptions.html) object passed in the `init` tracer's method.

Could we move this value (or at least the key of Verbosity - OFF, MANDATORY, INFORMATION, DEBUG) to the config object?

Reply:
moved to config object and now it's possible to configure it via tracer options